athlinks-ontology.ttl

```diff
@@ -470,3 +467,14 @@ schema:GenderType dc:creator "nextdude"^^xsd:string ;
               )
 ] .
 
+## Properties/Predicates
+
+### http://adp.athlinks.com/schema/hasEntry
+:hasEntry a owl:ObjectProperty ;
+  rdfs:domain schema:Person ;
+  rdfs:range athlinks:AthleticEventEntry .
+
+### http://adp.athlinks.com/schema/entryBelongsTo
+:entryBelongsTo a owl:ObjectProperty ;
```

Comment:
This is redundant I think. If a AthleticEvent is linked to a person via its competitor field; and the AthleticEventEntry is linked by its competitor field to the same person, then we can infer this backwards link and don't need it to get access to the other properties we put on the AthleticEventEntry object.

```
select ?event, ?athleteOrTeam, ?entry
where {
  ?event a athlinks:AthleticEvent;
       schema:competitor ?athleteOrTeam.
  ?entry a athlinks:AthleticEventEntry;
          schema:competitor ?athleteOrTeam.
}
```

Reply:
I think it makes more sense to have the `AthleticEventEntry` linked to the `AthleticEvent` via an `event` property (it could even be a [`sportsEvent`](http://meta.schema.org/sportsEvent) property). But as long as those property links exist and are uniform throughout the data model, I'm fine with re-using existing predicates when they aren't too general for our purposes.